tools/slang-test/options.cpp

```diff
@@ -290,9 +290,9 @@ static bool _isSubCommand(const char* arg)
         {
             optionsOut->skipApiDetection = true;
         }
-        else if (strcmp(arg, "-emit-spirv-directly") == 0)
+        else if (strcmp(arg, "-emit-spirv-via-glsl") == 0)
         {
-            optionsOut->emitSPIRVDirectly = true;
+            optionsOut->emitSPIRVDirectly = false;
         }
```

Comment:
Shouldn't we still support "-emit-spirv-directly"?
"tests/compute/half_texture.slang", as an example, still uses the option.

Reply:
We do, it just no longer has an effect because emit spirv directly is the default.